### ISSUE TITLE
[BUG] Buffering indicator (nearly) always visible

### ISSUE THREAD

**Issue by themadmrj** [jhomlala/betterplayer#237]
**Describe the bug**
The buffering circular progress indicator is visible on top on the video that is currently playing. Even with a short video that plays until the end and restarts in a loop, the progress indicator is still visible. Sometimes it dissapears, but I couldn't find any pattern to it.

Here is my config:

```
_betterPlayerController = BetterPlayerController(BetterPlayerConfiguration(
  autoDetectFullscreenDeviceOrientation: true,
  fit: BoxFit.cover,
  autoPlay: widget.position == ZapPosition.top ? true : false,
  looping: true,

  deviceOrientationsOnFullScreen: [DeviceOrientation.portraitUp],
  controlsConfiguration: BetterPlayerControlsConfiguration(
      enableOverflowMenu: false, enableSkips: false),
));
```

Here is a screenshot of the problem.

![Capture](https://user-images.githubusercontent.com/11475495/105029200-4cf6e880-5a52-11eb-8a4c-d30067da7a70.PNG)

**Comment by rahulkushwaha-techconfer** [comment]
I am fecing same issues

**Comment by jhomlala** [comment]
Please provide more details in issue ticket, don't skip issue template.

Fixed in 0.0.47.
